repo/meta/index.json

```diff
@@ -29,6 +72,22 @@
       }
     },
     {
```

Comment:
@jdef how the heck did you catch that so quickly?  @elingg catch it as well.   on it.   This is PR that is looking for agreement and approval still.  So even after it is technically correct... there may be some delay (I hope not much).  I expect it to happen this week.